django/db/migrations/operations/models.py

```diff
@@ -307,6 +307,25 @@ def state_forwards(self, app_label, state):
                 new_fields.append((name, field))
             state.models[related_key].fields = new_fields
             state.reload_model(*related_key)
+        # Repoint M2Ms with through pointing to us
+        related_models = {f.remote_field.model for f in model._meta.fields if getattr(f.remote_field, 'model', None)}
+        model_name = "%s.%s" % (app_label, self.old_name)
+        for related_model in related_models:
+            if related_model == model:
+                related_key = (app_label, self.new_name_lower)
```

Comment:
Alright let's keep it as it is then. I just wanted to make sure this case was covered by a test.